main.go

```diff
@@ -40,6 +41,8 @@ func main() {
 	flag.BoolVar(&flags.remove, "rm", false, "first remove output file, if it exists")
 	flag.BoolVar(&flags.withResets, "with-resets", false,
 		"generate functions to facilitate resetting calls made to a mock")
+	flag.BoolVar(&flags.writePkgComment, "write-pkg-comment", false,
+		"write package documentation comment (default false)")
```

Comment:
with the default value set to `false` means that the changes in this pull request do not affect the existing behavior.